### PR TITLE
QA8-1374 add from TC-053 to TC-061

### PR DESCRIPTION
**Pull request by andkr5150**

### CODE REVIEW

File: test/extended/gender.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import {sel} from "../../data/selectors";
+import {age, gender, name, story} from "../../data/testData";
+
+describe('Gender field suit', function () {
+
+    before('Open App', function () {
+        browser.url('');
+    });
+
+    describe('Placeholder', function (){
```
Comment (denispalkin) [inline test/extended/gender.js:10]: The title of describe 'Placeholder' is wrong here. It's the start of "Positive cases".

File: test/extended/gender.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        });
+
+        it('TC-055 Button "it" is enabled', function () {
+            let elem = $$(sel.requiredLabel)[2].isEnabled();
+            expect(elem).toEqual(true);
+        });
+
+        it('TC-056 User can choose only one button at the time : he', function () {
+            $$(sel.radioButtons)[gender.he].click();
+            const elem = $('.ant-radio-checked').isDisplayed();
```
Comment (denispalkin) [inline test/extended/gender.js:29]: Here and further `$('.ant-radio-checked')` is hardcoded, which is not good. It needs to be replaced with reference to the `selectors.js` file.

File: test/extended/gender.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            expect(elem).toEqual(true);
```
Comment (denispalkin) [inline test/extended/gender.js:30]: I think the logic of this test is wrong. Our spreadsheet for expected result says: "she" and "it" are grey, meaning they should be disabled `isEnabled() === false`

File: test/extended/gender.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        });
+
+        it('TC-057 User can choose only one button at the time : she', function () {
+            $$(sel.radioButtons)[gender.she].click();
+            const elem = $('.ant-radio-checked').isDisplayed();
+            expect(elem).toEqual(true);
```
Comment (denispalkin) [inline test/extended/gender.js:36]: I think the logic of this test is wrong. Our spreadsheet for expected result says: "he" and "it" are grey, meaning they should be disabled isEnabled() === false

File: test/extended/gender.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        });
+
+        it('TC-058 User can choose only one button at the time : it', function () {
+            $$(sel.radioButtons)[gender.it].click();
+            const elem = $('.ant-radio-checked').isDisplayed();
+            expect(elem).toEqual(true);
```
Comment (denispalkin) [inline test/extended/gender.js:42]: I think the logic of this test is wrong. Our spreadsheet for expected result says: "he" and "she" are grey, meaning they should be disabled isEnabled() === false

**Review by denispalkin — CHANGES_REQUESTED** [review]

File: test/extended/gender.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        it('TC-061 User can switch the option: it -> is Checked', function () {
+            $$(sel.radioButtons)[gender.it].click();
+            const elem = $$('.ant-radio-input')[2];
+            expect(elem).toBeChecked()
+        });
+
+//        browser.pause(2000);
+
+    });
+});
```
Comment (dalaran78) [inline test/extended/gender.js:66]: No new line in the end of file

**Review by dalaran78 — CHANGES_REQUESTED** [review]
Absolutely solidarity with our respected code keeper, master Denis :)

File: test/extended/gender.js (hunk 4 above)
Comment (MariiaSosedova) [reply test/extended/gender.js:36]: + Hardcoding + for these actions you need to create a function on the methods.js

File: test/extended/gender.js [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+            expect(elem).toBeChecked()
+        });
+
+        it('TC-061 User can switch the option: it -> is Checked', function () {
+            $$(sel.radioButtons)[gender.it].click();
+            const elem = $$('.ant-radio-input')[2];
+            expect(elem).toBeChecked()
+        });
+
+//        browser.pause(2000);
```
Comment (MariiaSosedova) [inline test/extended/gender.js:63]: Does it really need here?

**Review by MariiaSosedova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**